README.md

```diff
@@ -2,33 +2,651 @@
 
 > NextJS adapter with Guillotine support and basic views
 
-## Install
+## Installation
+
+---
 
 ```bash
 npm i --save @enonic/nextjs-adapter
 ```
 
-## Use
+## Usage
+
+---
 
-Just import required functions and types in your project:
+Just import required functions and views in your project:
 
 ```tsx
-import {fetchContent} from "@enonic/nextjs-adapter/guillotine/fetchContent";
+// all functions and interfaces are accessible at the library root
+import {fetchContent} from "@enonic/nextjs-adapter";
 
-// ...
+// views are placed in a folder called views and are default exports in files
+import StaticContent from "@enonic/nextjs-adapter/views/StaticContent"
 
+// ...
 const {page} = await fetchContent(path, context);
+// ...
+
+return <StaticContent/>
+```
+
+## API
+
+---
+
+### Functions
+
+They are available at the library root at `@enonic/nextjs-adapter`
+
+### <a id="fetch-content"></a>`fetchContent(contentPath: string | string[], context: Context) => Promise<FetchContentResult>`
+
+Fetches the content/component by path with component queries optimization, page structure as well as runtime info calculation. This is the
```

Comment:
Unnecessary linebreak?

Reply:
Doesn't affect the output, but easier to navigate the source